libc-bottom-half/cloudlibc/src/libc/time/nanosleep.c

```diff
@@ -5,6 +5,7 @@
 #include <errno.h>
 #include <threads.h>
 #include <time.h>
+#include <_/cdefs.h> // for __strong_reference
```

Comment:
Instead of adding this include, let's just delete the `thrd_sleep` alias here. If we want `thrd_sleep`, it's better to use the musl definition, which correctly implements the C11 behavior.